index.php

```diff
@@ -4,22 +4,23 @@
 
 <main id="main-content">
   <section id="posts" class="container">
-    <div class="grid-row">
+    <div id="shuffle-preloader"></div>
+    <div id="shuffle-container" class="grid-row">
 
 <?php
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
 ?>
-        <article <?php post_class('grid-item item-s-12 item-m-4'); ?> id="post-<?php the_ID(); ?>">
+        <article <?php post_class('shuffle-item item-s-12 item-m-4'); ?> id="post-<?php the_ID(); ?>">
```

Comment:
thats fine by me. I need to clarify how exactly the mobile adaption is supposed to happen as I only remember a kinda throwaway line about developer does mobile or responsive or something like that